content/doc/book/managing/casc.adoc

```diff
@@ -0,0 +1,287 @@
+and it provides some checks on the values that are provided.
+
+The JCasC configuration file can be checked into an SCM,
+which enables you to determine who made what modifications to the configuration
+and to roll back to a previous configuration if necessary.
+
+The
+link:https://plugins.jenkins.io/configuration-as-code[Configuration as Code] plugin
+must be installed on the Jenkins controller that you will use to build out your JCasC configuration.
+This plugin is not currently included in the suggested plugins list for Jenkins
+but the Docker image in the
+link:https://github.com/jenkinsci/docker[Docker] container
+includes the Configuration as Code plugin.
+It may also be
+link:https://github.com/jenkinsci/docker#preinstalling-plugins[pre-installed]
+on your instance.
+
+== Viewing the JCasC File
+
+When the Configuration as Code plugin is installed,
+you will see *Configuration as Code* in the *System Configuration* section
+of the *Manage Jenkins* page on your dashboard.
+Click on this link,
+then click on *View Configuration*
+to view the YAML file.
+
+This file is an export of the current configuration on this controller.
+It is *not* intended to offer a directly-usable Jenkins YAML configuration;
+it may be partial and may fail for some components.
+It does, however, provide a useful starting point.
+
+[NOTE]
+====
+The *Configuration as Code* UI page shows the full pathname of the YAML file being used
+and gives a box where you can specify a different file to use.
+See the information below about how to modify the location used for JCasC YAML files.
+====
+
+The default JCasC YAML file has four sections:
+
+* `jenkins` section defines the root Jenkins object,
+with configurations that can be set with the
+*Manage Jenkins >> Configure System*
+and *Manage Jenkins >> Configure Nodes and Clouds* screens.
+
+* `tool` section defines build tools that can be set on the
+*Manage Jenkins >> Global Tool Configuration* screen.
+
+* *unclassified* section definies miscellaneous configurations,
+including configuration for installed plugins.
+
+* `credentials` section defines credentials that can be set on the
+*Manage Jenkins >> Manage Credentials* screen.
+You may want to delete this section from your YAML file;
+this is discussed beginning at 27:53 in
+link:https://www.youtube.com/watch?v=ANU7jkxbZSM[How to Install Jenkins on CentOS 7 Using Ansible and JCasC].
+
+== YAML File Syntax
+
+JCasC defines the controller configuration in a YAML file.
+YAML is a popular serialization language for configuration information,
+with a syntax is straight-forward and easy to read but precise.
+
+Some key points about YAML syntax:
+
+* YAML files are case sensitive.
+* Indentation is very significant and specific.
+* Each item is a key/value pair.
+** The key is followed by a colon (:) and a space.
+** YAML converts certain strings into other types unless they are in quotes.
+*** true, false, Yes, No, et cetera are converted to Boolean values.
```

Comment:
```suggestion
*** `true`, `false`, `Yes`, `No`, et cetera are converted to Boolean values.
```
Not sur what "et cetera" can be

Reply:
Could have TRUE and True and maybe T.  But you are right -- "et cetera" is weak prose here.  I rewrote it to be parallel to the item about Booleans.